.buildkite/prepare-institutional-distribution.yml

```
@@ -0,0 +1,140 @@
+agents:
+  queue: "snapshots"
+
+env:
+  gs_bucket: gs://marinade-validator-bonds-mainnet
+  gs_bucket_institutional: gs://marinade-institutional-staking-mainnet
+  SLACK_API: https://slack.com/api/chat.postMessage
+  SLACK_FEED: "feed-institutional-staking"
+  CLAIM_TYPE: "institutional-distribution"
+
+steps:
+  - label: ":hammer_and_wrench: :rust: Build"
+    commands:
+    - '. "$HOME/.cargo/env"'
+    - 'cargo build --release --bin institutional-distribution-cli'
+    artifact_paths:
+      - target/release/institutional-distribution-cli
+
+  - input: "What is the epoch to generate settlements for?"
+    fields:
+      - text: "Epoch"
+        key: "epoch"
+        format: "[0-9]+"
+    if: "build.env('EPOCH') == null"
+    blocked_state: running
+
+  - wait: ~
+
+  - label: ":closed_lock_with_key: Concurrency gate lock"
+    command: echo "--> Start of concurrency gate"
+    concurrency_group: 'validator-bonds/prepare-institutional-distribution'
+    concurrency: 1
+
+  - wait: ~
+
+  - label: ":black_nib: Env variables setup"
+    commands:
+    - |
+      epoch=${EPOCH:-$(buildkite-agent meta-data get epoch)}
+      buildkite-agent meta-data set epoch "$$epoch"
+      echo "Epoch: '$$epoch'"
+
+  - wait: ~
+
+  - label: ":floppy_disk: :arrow_left: :cloud: Downloading Input JSON data"
+    commands:
+    - epoch=$(buildkite-agent meta-data get epoch)
+    - |
+      gcloud storage cp "$gs_bucket_institutional/$$epoch/institutional-payouts.json" "."
+    key: 'download-json'
+    artifact_paths:
+      - "./institutional-payouts.json"
+
+  - wait: ~
+
+  - label: ":scales: Prepare Validator Bonds Merkle Tree"
+    env:
+    commands:
+    - 'buildkite-agent artifact download --include-retried-jobs institutional-payouts.json .'
+    - 'chmod +x target/release/institutional-distribution-cli'
```

```suggestion
    - buildkite-agent artifact download --include-retried-jobs institutional-payouts.json .
    - chmod +x target/release/institutional-distribution-cli
```
I see the quotes removed in other pipelines, so suggesting here too for consistency.